config-map.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+data:
+  map-hash-bucket-size: "128"
```

Comment:
I kept this value because it is the default ConfigMap that I extended with our new values